docs/tutorials/scheduling.rst

```diff
@@ -353,7 +356,236 @@ and run the priority scheduler again.
     # To get a plot of the airmass vs where the blocks were scheduled
     plt.figure(figsize = (14,6))
     plot_schedule_airmass(priority_schedule)
+    plt.tight_layout()
     plt.legend(loc="upper right")
     plt.show()
 
-Nothing new shows up because Alpha Centauri isn't visible from APO.
\ No newline at end of file
+Nothing new shows up because Alpha Centauri isn't visible from APO.
+
+.. _scheduling-user_defined_schedulers:
+
+User-Defined Schedulers
+=======================
+
+There are many ways that targets can be scheduled with, only two of which
+are currently implemented. This example will walk through the steps for
+creating your own scheduler that will be compatible with the tools of
+the ``scheduling`` module.
+
+As you may have noticed above, the schedulers are assembled by making a
+call to the initializer of the class (e.g. `~astroplan.scheduling.PriorityScheduler`).
+Each of the schedulers is subclassed from the abstract `astroplan.scheduling.Scheduler`
+class, and our custom scheduler needs to be as well.
+
+For our scheduler, we will make one starts at the beginning and schedules
+the first ``ObservingBlock`` that it finds that doesn't have a score of zero: a
+``MinimumBarScheduler``. We need to include two methods, ``__init__`` and
+``_make_schedule`` for it to work:
+
+* The ``__init__`` is already defined by the super class, and accepts global constraints,
+  the `~astroplan.Observer`, the `~astroplan.scheduling.Transitioner`, a ``gap_time``,
+  and a ``time_resolution`` for spacing during the creation of the schedule.
+
+* It also needs a ``_make_schedule`` to do the heavy lifting. This takes a list of
+  `~astroplan.scheduling.ObservingBlock` objects and a `~astroplan.scheduling.Schedule`
+  object to input them into. This method needs to be able to check whether a
+  block can be scheduled in a given spot, and be able to insert it into the
+  schedule once a suitable spot has been found.
+
+Here's the ``MinimumBarScheduler`` implementation::
```

Comment:
What is the _bar_ in `MinimumBarScheduler`? Maybe `SimpleScheduler` would be good?